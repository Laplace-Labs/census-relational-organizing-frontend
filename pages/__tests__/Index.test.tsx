import { shallow } from 'enzyme';
import Index from '../Index';
import toJSON from 'enzyme-to-json';

describe('<Index/>', () => {
    it("Renders hello world", async () => {
        const wrapper = shallow(<Index/>);
        expect(toJSON(wrapper.find('h1'))).toMatchSnapshot();
    })
  
})