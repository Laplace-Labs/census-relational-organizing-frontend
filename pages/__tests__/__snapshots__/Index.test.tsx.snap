// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index/> Renders hello world 1`] = `
<h1>
   Hello World
</h1>
`;
